src/libm/sleefsimdsp.c

```diff
@@ -1818,6 +1818,9 @@ EXPORT CONST vfloat xfmaf(vfloat x, vfloat y, vfloat z) {
 static INLINE CONST vint2 vcast_vi2_i_i(int i0, int i1) { return vcast_vi2_vm(vcast_vm_i_i(i0, i1)); }
 
 EXPORT CONST vfloat xsqrtf_u05(vfloat d) {
+#ifdef ACCURATE_SQRT
+  return vsqrt_vf_vf(d);
+#else
```

Comment:
I would terminate the #ifdef here. the rest of the function is being removed by the compiler anyway if the macro is on.